### ISSUE TITLE
The effect after training is not as good as the pre training model you gave

### ISSUE THREAD

**Issue by liuxingyu123** [MengyangPu/RINDNet#5]
my use pytorch 1.7, gpu:1,  bachsize:4,  epochs:70  picture：[https://s3.bmp.ovh/imgs/2022/05/08/1549ef0c29221c39.png](url)

**Comment by MengyangPu** [comment]
Hello, we did not fix the seed during training, resulting in different random numbers.
